frontend/src/tests/FetchDeveloper.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { render, screen, waitFor } from '@testing-library/react';
+// imports toBeInTheDocument
+import '@testing-library/jest-dom';
+import { vi, describe, it, expect } from 'vitest';
+import axios from 'axios';
+import FetchDevelopers from '../components/FetchDevelopers';
+
+vi.mock('axios');
+describe('FetchDevelopers component', () => {
+  it('fetches and displays developers data', async () => {
+    // Mocked data returned by the Axios GET request
+    const developers = [
```

Comment:
...Seeing as we're using TypeScript :)
There is a certain `DeveloperInterface` that could be used here...

Reply:
I'm still getting used to Typescript, thanks for skilling me up!

If I'm not mistaken that would be a type, so I was thinking without searching it.
`developers:DeveloperInterface`??

Reply:
This would be an Array of DeveloperInterface so `developers: Array<DeveloperInterface>`

Reply:
I tried 😆 sounds good.